website/docs/r/vapp_firewall_rules.html.markdown

```diff
@@ -0,0 +1,146 @@
+* `destination_vm_nic_id` - (Optional) VM NIC ID to which this rule applies.
+* `source_port` - (Optional) The source port to match. Either a port number or `any`.
+* `source_ip` - (Optional) The source IP to match. Either an IP address, IP range or `any`.
+* `source_vm_id` - (Optional) Source VM identifier.
+* `source_vm_ip_type` - (Optional) The value can be one of: `assigned` - use assigned internal IP, `NAT` - use NATed external IP.
+* `source_vm_nic_id` - (Optional) VM NIC ID to which this rule applies.
+* `enable_logging`- (Optional) `true` value will enable rule logging. Default is `false`.
+
+## Importing
+
+~> **Note:** The current implementation of Terraform import can only import resources into the state.
+It does not generate configuration. [More information.](https://www.terraform.io/docs/import/)
+
+An existing an vApp network firewall rules can be [imported][docs-import] into this resource
+via supplying the full dot separated path to vapp network. An example is
+below:
+
+```
+terraform import vcd_vapp_firewall_rules.my-rules my-org.my-vdc.vapp_name.network_name
+```
+or using IDs:
+```
+terraform import vcd_vapp_firewall_rules.my-rules my-org.my-vdc.vapp_id.network_id
+```
+
+NOTE: the default separator (.) can be changed using Provider.import_separator or variable VCD_IMPORT_SEPARATOR
+
+[docs-import]:https://www.terraform.io/docs/import/
+
+After that, you can expand the configuration file and either update or delete the vApp network rules as needed. Running `terraform plan`
+at this stage will show the difference between the minimal configuration file and the vApp network rules stored properties.
+
+### Listing vApp Network IDs
+
+If you want to list IDs there is a special command **`terraform import vcd_vapp_firewall_rules.imported list@org-name.vcd-name.vapp-name`**
+where `org-name` is the organization used, `vdc-name` is VDC name and `vapp-name` is vApp name. 
+The output for this command should look similar to the one below:
+
+```shell
+$ terraform import vcd_vapp_firewall_rules.imported list@org-name.vdc-name.vapp-name
+vcd_vm_internal_disk.imported: Importing from ID "list@org-name.vdc-name.vapp-name"...
+Retrieving all vApp networks by name
+No	vApp ID                                                 ID                                      Name	
+--	-------                                                 --                                      ----	
+1	urn:vcloud:vapp:77755b9c-5ec9-41f7-aceb-4cf158786482	0027c6ae-7d59-457e-b33e-a89e97f0bdc1	Net2
+2	urn:vcloud:vapp:77755b9c-5ec9-41f7-aceb-4cf158786482	36986073-8051-4f6d-a1c6-bda648bdf6ba	Net1      		
+
+Error: resource was not imported! resource id must be specified in one of these formats:
+'org-name.vdc-name.my-independent-disk-id' to import by rule id
+'list@org-name.vdc-name.my-independent-disk-name' to get a list of disks with their IDs
```

Comment:
independent-disk? There are multiple places in the output of list command where left-over disk copy/pastes are. Please re-review whole output of the list command:

```
lvirbalas$ terraform import vcd_vapp_firewall_rules.vapp_fw list@demo-org.demo-vdc.demo-web
vcd_vapp_firewall_rules.vapp_fw: Importing from ID "list@demo-org.demo-vdc.demo-web"...
Retrieving all vApp networks by name
No	vApp ID							ID					Name
--	-------							--					----
1	urn:vcloud:vapp:2521e464-beec-4226-9fcf-3b37b240076c	07c10396-cd28-4c0b-a72a-c4d4f4697697	demo-net-web
2	urn:vcloud:vapp:2521e464-beec-4226-9fcf-3b37b240076c	10855c07-6685-4635-81ff-c5bb8851a852	demo-vapp-net-w-org
3	urn:vcloud:vapp:2521e464-beec-4226-9fcf-3b37b240076c	e836f48e-0172-49b3-88b6-f57e5d567567	demo-vapp-net-isolated

Error: resource was not imported! resource id must be specified in one of these formats:
'org-name.vdc-name.my-independent-disk-id' to import by rule id
'list@org-name.vdc-name.my-independent-disk-name' to get a list of disks with their IDs
```

Reply:
fixed

Reply:
Here:
<img width="876" alt="Screenshot 2020-06-18 at 19 25 49" src="https://user-images.githubusercontent.com/11330689/85047349-58232680-b19a-11ea-848f-92c58f2de427.png">
Second example should be formatted same way as the first one.

Reply:
![image](https://user-images.githubusercontent.com/41415459/85097157-50499d80-b1ff-11ea-940f-3f56f50c5a43.png)
done